pylint/pyreverse/diagrams.py

```diff
@@ -28,59 +43,73 @@ def __init__(self, from_object, to_object, relation_type, name=None):
 class DiagramEntity(Figure):
     """A diagram object, i.e. a label associated to an astroid node."""
 
-    def __init__(self, title="No name", node=None):
+    default_shape = ""
+
+    def __init__(self, title: str = "No name", node: NodeNG = None) -> None:
```

Comment:
`Pylance`/`pyright` warns about this because `None != NodeNG`. Perhaps:

```suggestion
    def __init__(self, title: str = "No name", node: NodeNG | None = None) -> None:
```

Reply:
Thanks for catching that. I thought `node: NodeNG = None` would be implicitly equivalent to `node: Optional[NodeNG] = None` (and therefore `NodeNG | None`), but I am also not a typing expert. So these PRs are a good practice for me. 😁

Reply:
Yeah, for some reason `mypy` doesn't complain about it but it doesn't hurt to be explicit.